osu.Game/Scoring/ScoreImporter.cs

```diff
@@ -56,10 +56,14 @@ public ScoreImporter(RulesetStore rulesets, Func<BeatmapManager> beatmaps, Stora
                 {
                     Logger.Log($@"Score '{archive.Name}' failed to import: no corresponding beatmap with the hash '{notFound.Hash}' could be found.", LoggingTarget.Database);
 
-                    // In the case of a missing beatmap, let's attempt to resolve it and show a prompt to the user to download the required beatmap.
-                    var req = new GetBeatmapRequest(new BeatmapInfo { MD5Hash = notFound.Hash });
-                    req.Success += res => PostNotification?.Invoke(new MissingBeatmapNotification(res, archive, notFound.Hash));
-                    api.Queue(req);
+                    if (!parameters.Batch)
```

Comment:
Just to make sure, since it's not mentioned anywhere: the batch flag is also set when importing 10 or more scores at a time, so in such a case the notifications will not show either. Are we okay with this? I could be, asking mostly for clarification.